--- a/docs/conf.py
+++ b/docs/conf.py
@@ -47,7 +47,6 @@
     'sphinx.ext.intersphinx',
     'sphinx.ext.todo',
     'sphinx.ext.viewcode',
-    "sphinx_ansible_theme",
 ]
 
 # Add any paths that contain templates here, relative to this directory.
@@ -83,7 +82,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_ansible_theme'
+#html_theme = 'sphinx_ansible_theme'
 html_title = "Ansible Runner Documentation"
 
 # Theme options are theme-specific and customize the look and feel of a theme
